pkg/identity/keystone/keystone.go

```diff
@@ -246,7 +246,8 @@ func (k *KeystoneAuth) Handler(w http.ResponseWriter, r *http.Request) {
 		userInfo := k.authenticateToken(w, r, token, data)
 
 		// Do synchronization
-		if k.syncer.syncConfig != nil && userInfo != nil {
+		// In the case of unscoped tokens, when project id is not defined, we have to skip this part
+		if k.syncer.syncConfig != nil && userInfo != nil && userInfo.Extra["alpha.kubernetes.io/identity/project/id"][0] != "" {
```

Comment:
How about the following codes?
if k.syncer.syncConfig != nil && userInfo != nil && len(userInfo.Extra["alpha.kubernetes.io/identity/project/id"]) != 0 {

Reply:
Done. Thanks!